src/schemas/Polymorphic.js

```diff
@@ -45,6 +45,11 @@ export default class PolymorphicSchema {
       return value;
     }
     const schema = this.isSingleSchema ? this.schema : this.schema[value.schema];
-    return unvisit(value.id || value, schema, entities);
+    let idOrValue;
+    if (typeof schema.getId === 'function') {
```

Comment:
Normalization doesn't add id to entity. I'm coming from roundtrip point of view.
```
  it('can denormalize previously normalized data', () => {
    const mySchema = new schema.Entity('cats', {}, { idAttribute: 'meow' });
    const cat = { 'meow': 'Meow', hair: 'none' };
    const normalizedCat = normalize(cat, mySchema);
    expect(normalizedCat).toMatchSnapshot();
    const denormalizedCat = denormalize(cat, mySchema, normalizedCat.entities);
    expect(denormalizedCat).toEqual(cat);
    expect(denormalizedCat).toMatchSnapshot();
  });
```